src/main/kotlin/no/nav/syfo/personhendelse/PersonhendelseService.kt

```diff
@@ -0,0 +1,58 @@
+package no.nav.syfo.personhendelse
+
+import no.nav.person.pdl.leesah.Endringstype
+import no.nav.person.pdl.leesah.Personhendelse
+import no.nav.syfo.application.database.DatabaseInterface
+import no.nav.syfo.domain.PersonIdentNumber
+import no.nav.syfo.oppfolgingstilfelle.OppfolgingstilfelleService
+import no.nav.syfo.personhendelse.db.createPerson
+import no.nav.syfo.personhendelse.db.getDodsdato
+import no.nav.syfo.util.kafkaCallId
+import org.slf4j.LoggerFactory
+import java.util.UUID
+
+class PersonhendelseService(
+    private val database: DatabaseInterface,
+    private val oppfolgingstilfelleService: OppfolgingstilfelleService,
+) {
+    suspend fun handlePersonhendelse(personhendelse: Personhendelse) {
+        if (personhendelse.doedsfall != null && personhendelse.endringstype == Endringstype.OPPRETTET) {
+            log.info("Received personhendelse: Hendelseid: ${personhendelse.hendelseId}, Endringstype: ${personhendelse.endringstype.name}, Oppltype: ${personhendelse.opplysningstype}, Dato: ${personhendelse.doedsfall?.doedsdato}")
+            personhendelse.personidenter
+                .mapNotNull { personIdent ->
+                    try {
+                        if (personIdent.isNullOrEmpty() || personIdent.length == 13) {
```

Comment:
Vil man ikke ha warning hvis identen er tom? 🤔

Reply:
Dette er jo mest for å unngå NullPointerException og at hele consumer'en krasjer (det er omtrent samme kode i IdenthendelseService'ene rundt omkring): Siden vi leser dette fra Kafka/Avro er jeg litt usikker på i hvilken grad vi kan stole på at null-verdier ikke forekommer siden det er litt på siden av det vanlige Kotlin-regimet. Men det er vel ikke noen grunn til å tro at PDL legger ut null-verdier her. I alle fall så kan vi ikke bruke sånne data til noe.